router/batchrouter/asyncdestinationmanager/common/common.go

```diff
@@ -112,10 +111,10 @@ type AsyncFailedPayload struct {
 }
 
 type FetchUploadJobStatus struct {
-	FailedJobsURL  string
-	Parameters     stdjson.RawMessage
-	ImportingList  []*jobsdb.JobT
-	OutputFilePath string
+	FailedJobInfo      string
```

Comment:
why we need both FailedJobURLs and PollResultFileURLs?

Reply:
because poll operation says if it has failed with errors, and in that response only we get the URL of the file

Reply:
from usage point of view how different they are?

Reply:
yeah we can use one